conan/cli/formatters/audit/vulnerabilities.py

```diff
@@ -109,11 +108,10 @@ def wrap_and_indent(txt, limit=80, indent=2):
                   "the returned vulnerabilities to conan-research@jfrog.com.\n",
                   fg=Color.BRIGHT_GREEN)
 
-
-def json_vuln_formatter(result):
-    response, errors_in_response = result
-    if not errors_in_response or response["data"]:
-        cli_out_write(json.dumps(response, indent=4))
+def json_vuln_formatter(result: dict):
+    errors_in_response = "conan_error" in result
+    if not errors_in_response or result["data"]:
```

Comment:
The json output doesn't need to be inhibited. It can be outputted unconditionally.
Then, after the output, it will raise an error.

Reply:
Done, commit [5e0c46a](https://github.com/conan-io/conan/pull/18160/commits/5e0c46a4f219d755f88b9a435543db0b5a8f5382)

Reply:
Fixed by #18171